src/Avalonia.Controls/MaskedTextBox.cs

```diff
@@ -32,7 +32,9 @@ public class MaskedTextBox : TextBox, IStyleable
              AvaloniaProperty.Register<MaskedTextBox, string?>(nameof(Mask), string.Empty);
 
         public static new readonly StyledProperty<char> PasswordCharProperty =
-             AvaloniaProperty.Register<MaskedTextBox, char>(nameof(PasswordChar), '\0');
+#pragma warning disable AVP1013 // AvaloniaProperty owners should not be added superfluously
+            TextBox.PasswordCharProperty.AddOwner<MaskedTextBox>();
```

Comment:
I don't think we even need this property here. Just OverrideDefault. Needs to be double checked.